lib/configs/react.ts

```diff
@@ -13,6 +13,7 @@ export = {
     'testing-library/no-dom-import': ['error', 'react'],
     'testing-library/no-node-access': 'error',
     'testing-library/no-promise-in-fire-event': 'error',
+    'testing-library/no-unnecessary-act': 'error',
```

Comment:
I think we should enable it with `isStrict` option set to true. What do you think?